monkey/monkey_island/cc/services/reporting/exploitations/monkey_exploitation.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from dataclasses import dataclass
+from typing import List
+
+from monkey_island.cc.database import mongo
+from monkey_island.cc.services.node import NodeService
+from monkey_island.cc.services.reporting.issue_processing.exploit_processing.exploiter_descriptor_enum import (  # noqa: E501
+    ExploiterDescriptorEnum,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class MonkeyExploitation:
+    label: str
+    ip_addresses: List[str]
+    domain_name: str
+    exploits: List[str]
+
+
+def get_monkey_exploited() -> List[MonkeyExploitation]:
```

Comment:
This needs unit tests.

Reply:
As discussed, hard to add proper unit tests without a heavier refactor and let's keep PR's one refactoring at a time